Documentation/ReleaseNotes.md

```diff
@@ -118,6 +118,10 @@ See [ARFoundation](CrossPlatform/UsingARFoundation.md#install-required-packages)
 
 Some asmdef files are changed and are now only supporting Unity 2018.4.13f1 or later. Compilation erros will show up when upating to MRTK 2.5 in earlier versions of Unity. This can be fixed by going to `Assets\MRTK\Providers\XRSDK\Microsoft.MixedReality.Toolkit.Providers.XRSDK.asmdef` in the project window and removing the missing reference in the inspector. Repeat those steps with `Assets\MRTK\Providers\Oculus\XRSDK\Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef` and `Assets\MRTK\Providers\WindowsMixedReality\XRSDK\Microsoft.MixedReality.Toolkit.Providers.XRSDK.WMR.asmdef`. Note you must revert the changes by replacing those three asmdef files with original (i.e. unmodified) ones when upgrading to Unity 2019.
 
+**CameraCache may create a new camera on shutdown**
+
+In some specific situations, it is possible for the CameraCache to re-create the MainCamera on shutdown. Please see [this issue](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8459) for more information.
```

Comment:
Seems like the issue is mostly specific to leap right? Might be worth calling out the context in which it happens.

Also as @MaxWang-MS  calls out, it's probably better in the known issues.

This isn't actually a breaking change - it's not a regression or related AFAIK to the 2.5 payload.

Reply:
thanks folks! I will make the change now

Reply:
both done!